### DISCUSSION TITLE
Too stupid to modify REBAR

### DISCUSSION THREAD

**Discussion by x1091** [xCuri0/ReBarUEFI#168] [General]
Can anyone plese help me, i need rebar to work because i just ordered an ARC gpu.
I tried modifying the bios, but the filetype changed to "rom", and i cannot flash such file extensions, the original bios has a "cap" extension.
I uploaded the original and the modified BIOS.
I must mention the modified bios has just the rebar file added and not anything else from the guide.
  
Original

[H110M-R-ASUS-4210.zip](https://github.com/xCuri0/ReBarUEFI/files/14431618/H110M-R-ASUS-4210.zip)

Modified
[H110M-R-ASUS-4210 with REBAR.zip](https://github.com/xCuri0/ReBarUEFI/files/14431625/H110M-R-ASUS-4210.with.REBAR.zip)

**Comment by MrAnonymous7122** [comment]
When saving from UEFITool, set the file type as BIOS files and then set the name as BIOS.CAP.
